articles/quickstart/spa/angular/interactive.md

```diff
@@ -83,7 +83,7 @@ Sorry about that. Here are a few things to double check:
 - make sure you configured the correct `authorizationParams.redirect_uri`
 - make sure you added the `LoginButtonComponent` button to the module's declarations
 
-Still having issues? To get more help, check out our [documentation](/docs) or visit our [community page](https://community.auth0.com).
+Still having issues? To get more help, check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com).
```

Comment:
Does this still happen if you use `/` instead of `/docs`? The docs app seems to be mounted under `/docs`, so `/` will actually be `/docs`.

Reply:
I did think to use `/` but after checking other quickstarts they all used `[documentation](https://auth0.com/docs)` so I went with that.

Do the interactive quickstarts get consumed/embedded by any other sites at all like the tutorial quickstarts? If we used `/` would that potentially cause issues there?

Reply:
I would recommend sticking to relative, see https://github.com/auth0/docs/pull/10227#discussion_r1115650033